editor/src/components/inspector/common/property-path-hooks.ts

```diff
@@ -376,31 +375,49 @@ function parseFinalValue<PropertiesToControl extends ParsedPropertiesKeys>(
 ): {
   finalValue: ParsedPropertiesValues
   isUnknown: boolean
+  usesComputedFallback: boolean
 } {
   const valueAsMaybe = eitherToMaybe(simpleValue)
   const rawValueAsMaybe = eitherToMaybe(rawValue)
 
-  function finalValueFromReal(): ParsedPropertiesValues {
-    if (realValue == null) {
-      const parsedComputedValue = parseAnyParseableValue(property, computedValue, null)
-      return defaultEither(emptyValues[property], parsedComputedValue)
-    } else {
-      const parsedRealValue = parseAnyParseableValue(property, realValue, null)
-      return defaultEither(emptyValues[property], parsedRealValue)
+  function finalValueFromReal(
```

Comment:
this is very hard to read :|

Reply:
I think extracting the return type would help since it's used above too

Reply:
fixed in effda55